widget/ui/src/components/ListItem/ListItem.types.ts

```diff
@@ -1,7 +1,7 @@
 import type { CSSProperties } from '@stitches/react';
 
 export type ListItemProps = {
-  title?: string | React.ReactElement;
+  title: string | React.ReactElement;
```

Comment:
The ui package doesn't build correctly. Please check

Reply:
fixed